session/session.go

```diff
@@ -269,20 +296,43 @@ func (s *Session) onACK(surbID *[constants.SURBIDLength]byte, ciphertext []byte)
 
 	switch msg.SURBType {
 	case cConstants.SurbTypeACK:
-		// XXX TODO fix me
+		// Validate that the ACK payload is entirely '0x00'.
```

Comment:
I think we may soon get rid of "ACKS"... i mean instead we will do everything with "service queries".

Reply:
a multispool service reply will be equivalent to an ACK